Networking/NetworkingTests/Testing/XCTestCase+Wait.swift

```diff
@@ -7,18 +7,37 @@ extension XCTestCase {
     /// Example usage:
     ///
     /// ```
-    /// waitForExpectation { expectation in
+    /// waitForExpectation(timeout: TimeInterval(10)) { expectation in
     ///     doSomethingInTheBackground {
     ///         expectation.fulfill()
     ///     }
     /// }
     /// ```
     ///
-    func waitForExpectation(description: String? = nil,
-                            timeout: TimeInterval = Constants.expectationTimeout,
-                            _ block: (XCTestExpectation) -> ()) {
+    public func waitForExpectation(description: String? = nil,
+                                   timeout: TimeInterval,
+                                   _ block: (XCTestExpectation) -> ()) {
         let exp = expectation(description: description ?? "")
         block(exp)
         wait(for: [exp], timeout: timeout)
     }
+
+    /// Creates an XCTestExpectation and waits for `block` to call `fulfill()`.
+    ///
+    /// Example usage:
+    ///
+    /// ```
+    /// waitForExpectation { expectation in
+    ///     doSomethingInTheBackground {
+    ///         expectation.fulfill()
+    ///     }
+    /// }
+    /// ```
+    ///
+    public func waitForExpectation(description: String? = nil,
+                                   _ block: (XCTestExpectation) -> ()) {
```

Comment:
It did. It stopped working when I changed this to `public` so we can use it in the WooCommerce package. 

This should really be in that `Shared.framework` we talked about last time. Would you like to prototype and submit a proposal for that, @jaclync? 😁  I still have Copiable in my backlog so it'll probably be a while before I can get to that. 😐

Reply:
FYI! I was wrong about this being useable in Yosemite. It's not because it's in a different target (NetworkingTests). 🤦 It looks like we'd need something like a SharedTests.framework to share test utilities. 

I'll fix this mess in an upcoming PR.